component/component.go

```diff
@@ -125,3 +125,9 @@ type HTTPComponent interface {
 	// will receive a request to just `/metrics`.
 	Handler() http.Handler
 }
+
+// SubgraphOwner is the component calling the SubgraphHandler.
+type SubgraphOwner interface {
+	ID() string
+	IDs() []string
+}
```

Comment:
Could we add a docstring about what the methods mean? Is it the ID of the parent and the IDs of its children?

Also, is this the `DelegateComponent` interface we reference in a comment below?